### DISCUSSION TITLE
webh5同一个页面播放多个pagView,浏览器和安卓机都没有问题，在苹果手机上访问h5页面的时候造成页面渲染特别卡顿，点任何地方都没有反应

### DISCUSSION THREAD

**Discussion by qss66** [Tencent/libpag#2056] [Q&A]
![wecom-temp-75891-16457b637ab715750edf032423ca8c8b](https://github.com/Tencent/libpag/assets/58547630/c18d4086-8bc6-444a-8d9c-bb200f1355b2)

**Comment by zhalice2011** [comment]
你好, 请问有啥报错信息吗, 方便提供可复现 demo 以及机型信息吗

**Comment by CodeJhF** [comment]
感谢 @zhalice2011 的回复，也抱歉我们没有及时跟进这条。

如果问题仍存在，请在最新版本（4.4 / 4.5）上复现并补充版本号与 demo，我们会优先跟进，本条先关闭。（stale-close）
